flocker/acceptance/test/test_testtools.py

```diff
@@ -0,0 +1,131 @@
+# Copyright ClusterHQ Ltd.  See LICENSE file for details.
+
+"""
+Tests for flocker.acceptance.testtools.
+"""
+
+from eliot import ActionType
+from eliot.testing import (
+    assertContainsFields,
+    assertHasAction,
+    capture_logging,
+)
+from twisted.internet.defer import succeed
+from twisted.trial.unittest import SynchronousTestCase
+
+from ..testtools import (log_method, _ensure_encodeable)
+
+
+class EnsureEncodeableTests(SynchronousTestCase):
+    """
+    Tests for ``_ensure_encodeable``.
+
+    ``_ensure_encodeable`` is an implementation detail of ``log_method``, so
+    these tests are not strictly necessary. They were written because we were
+    seeing unexpected behavior and wanted to make sure that
+    ``_ensure_encodeable`` wasn't the cause.
+    """
+
+    def test_encodeable(self):
+        """
+        If given an encodeable object, _ensure_encodeable just returns it.
```

Comment:
Any object can be encoded somehow. Make it clear that this means that it can be encoded by Eliot, or by JSON.